python/ray/tune/ray_trial_executor.py

```diff
@@ -216,6 +218,12 @@ def in_staging_grace_period(self) -> bool:
         return self._pg_manager.in_staging_grace_period()
 
     def set_max_pending_trials(self, max_pending: int) -> None:
+        if len(self._cached_actor_pg) > 0:
+            logger.warning(
+                "Cannot update maximum number of queued actors for reuse "
+                "during a run.")
```

Comment:
What does this provide the user? I'm thinking that if I see this error message I won't know what to do.

Reply:
Agree, that's not great - in this case it might be acceptable as this API is not used in a regular Ray Tune workflow, but only when a custom trial runner is used (or unit tested in tests).